src/pages/projects/forms/ProjectDetailsForm.tsx

```diff
@@ -226,7 +226,7 @@ const ProjectDetailsForm: FC<Props> = ({ formik }) => {
                   )
                 }
                 checked={formik.values.features_networks_zones}
-                disabled={formik.values.readOnly}
+                disabled={isReadOnly || isDefaultProject}
```

Comment:
Actually, I tried disabling the network zones on a non-empty project and this is what I get:

![image](https://user-images.githubusercontent.com/56583786/230425781-8efe732f-37b2-4ac5-80b8-23cf17595e2e.png)

Reply:
I amended the commit, so we disable the network zones when it is active and the project is non-empty.